src/Database/Adapter/Mongo.php

```
@@ -0,0 +1,2216 @@
+     *
+     * @param string $name
+     * @param array<string, mixed> $document
+     *
+     * @return array<string, mixed>
+     * @throws Duplicate
+     */
+    private function insertDocument(string $name, array $document): array
+    {
+
+        try {
+            $bla = $this->client->insert($name, $document);
```

PHPMD (2.15.0)

Avoid unused local variables such as '$bla'. (Unused Code Rules)

(UnusedLocalVariable)